plane/source/docs/takeoff-landingpage.rst

```diff
@@ -0,0 +1,55 @@
+.. _takeoff-landingpage:
+
+=======
+Takeoff
+=======
+
+Taking the plane off can be accomplished in many ways: Rolling Takeoff, Hand Launch, Bungee, or VTOL (Verical Takeoff and Landing) if a :ref:`quadplane-support`. And it can be done manually or automatically.
+
+Rolling Takeoff
+===============
+
+In this case, the plane has landing gear with steerable nose gear, usually coupled to the rudder control and the plane is usually launched from a smooth grass or paved surface. Once aligned nose into the wind, throttle is gradually increased to takeoff value (usually full throttle) while the aircraft direction is controlled by rudder. Once sufficient airspeed to lift off is obtained, slight up elevator will rotate the nose to climb attitude.
+
+MANUAL or FBWA modes are most commonly used during takeoffs.
+
+For automatic takeoffs see :ref:`automatic-takeoff` section.
+
+Hand Launch
+===========
+
+Many planes do not have landing gear, or a smooth runway is not available. Hand launching is a very common takeoff method, but also can be very frustrating if not setup and executed properly.
+
+The keys to a successful hand launch are :
+
+- :ref:`Proper CG <guide-center-of-gravity>` 
+- Having flying surfaces centered and neutral at neutral RC stick positions
+- Having the "level" calibration setup such that the level horizon (0 deg pitch) in FBWA mode results in the wing chord line being a few degrees positive, which will result in a positive angle of attack in "level" flight, the normally cruising attitude for a plane.
+- Using FBWA mode or :ref:`automatic-takeoff` . While launching in MANUAL can be done, using a stabilized mode will eliminate a lot of pilot effort to overcome trim issues (see Tip at end of this page). If using FBWA mode, a few degrees of up elevator (~10%) during the launch will result in an automatic climb. Add this via a switch or stick pressure. Do not change TX trim positions used during RC calibration since it affects all other modes of flight.
```

Comment:
Suggest more up elevator; I launch my planes at FULL (100% and 30-degrees pitch) elevator, and that keeps them from diving before climbing... a common problem.

Reply:
the problem is that if CG is a bit back, or the throw is weak, this almost always results in a stall....your planes must tend to be a bit nose heavy....since this is aimed at a newbie (experienced users dont need this advice), it is safer to have a grass burner launch (if they are very nose heavy) than a tip rolling stall...the former almost never damages, the latter is a potential rebuild....

Reply:
The wiki already warns the user about being tail-heavy during setup; I don't think we need to compromise the success of most by thinking a few users will be tail-heavy. Most aircraft today already have very conservative manufacturer CG's. 

Furthermore, CG location actually doesn't have nearly as much of an effect on takeoff stability as one might think. The aerodynamics at play are minimal because we're at an airspeed so low that stability doesn't really come into play. We also often have thrust over an elevator because the throttle is full (unlike a power-off stall/spin).

Saying the wiki is primarily for the novice user isn't really great; I'm a seasoned user and I still get great advice by overlooking the wiki. One of the most frustrating things is being an advanced RC pilot with a plane that is set up very well in Manual, but the first hand launch skims the grass and crashes an otherwise perfect plane. Those people need the most specific and accurate advice, and they are definitely important subjects for this wiki.

I have had a tip rolling stall on takeoff before, but that was because I threw the tail sideways and down. It had nothing to do with any demand from the autopilot; the control surfaces could have been reversed and it would have done the same thing. Only difference is that it would have nearly recovered on its own if it wasn't so close to the ground. Bad throw technique causes lots of problems.

Reply:
I agree with Henry on this